src/Qir/Runtime/prerequisites.ps1

```diff
@@ -6,9 +6,10 @@
 if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     if (($IsWindows) -or ((Test-Path Env:/AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Win")))) {
         if (!(Get-Command clang        -ErrorAction SilentlyContinue) -or `
-            !(Get-Command clang-format -ErrorAction SilentlyContinue)) {
-            choco install llvm --version=11.1.0
-            Write-Host "##vso[task.setvariable variable=PATH;]$Env:Path;C:\Program Files\LLVM\bin"
+            !(Get-Command clang-format -ErrorAction SilentlyContinue) -or `
+            (Test-Path Env:/AGENT_OS)) {
+            choco install llvm --version=11.1.0 --allow-downgrade
+            Write-Host "##vso[task.setvariable variable=PATH;]C:\Program Files\LLVM\bin;$Env:Path"
```

Comment:
Just in case I would recommend the following:
```suggestion
            Write-Host "##vso[task.setvariable variable=PATH;]C:\Program Files\LLVM\bin;$Env:PATH"
```
(it makes no difference for Win, but some time ago I copy-pasted the fragment `$Env:Path` to the scope common for all platforms, and on Linux the behavior was unexpected)